catroid/res/values/strings.xml

```diff
@@ -621,6 +621,10 @@
     <string name="formula_editor_function_arctan_parameter">(0)</string>
     <string name="formula_editor_function_exp">exp</string>
     <string name="formula_editor_function_exp_parameter">(0)</string>
+    <string name="formula_editor_function_floor">floor</string>
+    <string name="formula_editor_function_floor_parameter">(0)</string>
+    <string name="formula_editor_function_ceil">ceil</string>
+    <string name="formula_editor_function_ceil_parameter">(0)</string>
```

Comment:
Should be 0.3 to illustrate how it can be used. 0.3 is better here since when tapping on compute, the user immediately sees the difference to round. Also please for the German translation.